### ISSUE TITLE
Scheduler rearrange logic could be more optimized

### ISSUE THREAD

**Issue by Narflex** [google/sagetv#51]
There's a case we don't handle in our Scheduler rearrange logic that would be better handled there than before we went to the exhaustive search.  We don't try to shift an existing scheduled recording over to another tuner with an open slot in order to make room for a new one. We will swap the recording time if it has other airings...but that's different. I'm not 100% sure this affects systems that have all tuners with the same lineup...but I think it could.  SageTV has always had this deficiency..this is nothing new. :)

**Comment by CraziFuzzy** [comment]
So this would require checking if more than one tuner offers that airing, and if so, try a different one for that airing, and recheck the conflicts?  There's have to be a way to track which of the available tuners have been checked with this file in this iteration... Might be.. messy.

**Comment by Narflex** [comment]
Yeah, this is far from an easy change to do...it was just on my list of things to fix that's been sitting there for years so figured I'd add it to the tracker. :)
